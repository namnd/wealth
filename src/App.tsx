import React from 'react';

export default function App() {
    return <button className="btn mr-2 mt-6" type="button">Click here to start</button>;
}
