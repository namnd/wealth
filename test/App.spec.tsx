import React from "react"
import { shallow } from "enzyme"
import { shallowToJson } from "enzyme-to-json"
import App from "../src/App"

describe('App', () => {
    it('should render correctly', () => {
        const output = shallow(<App />)

        expect(shallowToJson(output)).toMatchSnapshot()
    });
});
