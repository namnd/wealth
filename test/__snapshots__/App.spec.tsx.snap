// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<button
  className="btn mr-2 mt-6"
  type="button"
>
  Click here to start
</button>
`;
